Source/WebKit2/Shared/WebPreferencesDefinitions.h

```diff
@@ -110,7 +110,7 @@
     macro(JavaEnabledForLocalFiles, javaEnabledForLocalFiles, Bool, bool, false, "", "") \
     macro(OfflineWebApplicationCacheEnabled, offlineWebApplicationCacheEnabled, Bool, bool, false, "", "") \
     macro(LocalStorageEnabled, localStorageEnabled, Bool, bool, true, "", "") \
-    macro(DatabasesEnabled, databasesEnabled, Bool, bool, true, "", "") \
```

Comment:
This should default to true, and WKPreferencesSetDatabasesEnabled() should be called instead to disable the preference.